Sources/Sentry/SentrySession.m

```diff
@@ -238,6 +243,10 @@ - (void)incrementErrors
 
         [serializedData setValue:_distinctId forKey:@"did"];
 
+        if (_abnormalMechanism != nil) {
+            serializedData[@"abnormal_mechanism"] = _abnormalMechanism;
```

Comment:
Where is this actually set? The only place I see the `abnormalMechanism` ivar or property being set is in the initWithJSONObject method. I expected this to be set from something like the ANR tracking integration.

Reply:
Oh I'm deeply sorry. I should have clarified this in the PR description. This isn't used yet. I will glue this together in another PR.